Release/include/pplx/pplxtasks.h

```diff
@@ -2708,7 +2708,7 @@ class task_completion_event
     {
         // Subsequent sets are ignored. This makes races to set benign: the first setter wins and all others are
         // ignored.
-        if (_IsTriggered())
+        if (_IsTriggered() || _M_Impl->_HasUserException())

```

Comment:
_HasUserException() is not safe to call from multiple threads if someone could be writing over that.